piedpiper/gman/exceptions.py

```diff
@@ -0,0 +1,12 @@
+class TaskError(Exception):
+    def __init__(self, *args, **kwargs):
+        Exception.__init__(self, *args, **kwargs)
+
+
+class TaskFailed(TaskError):
```

Comment:
are you going to have more child classes of TaskError... otherwise this seems like it should just be TaskError

Reply:
Thought was to yes, eventually have more child classes of TaskError. I also needed a way to differentiate between len(events_failed) (we actually saw "failed" in GMan VS TaskError() after we hit our retry_max (we just timed out). Maybe we should raise a different exception here though..  TimeoutError?